src/editor/BspRenderer.cpp

```diff
@@ -177,10 +177,13 @@ void BspRenderer::loadTextures() {
 		}
 
 		COLOR3* imageData = new COLOR3[tex.nWidth * tex.nHeight];
-		int sz = tex.nWidth * tex.nHeight;
 
-		for (int k = 0; k < sz; k++) {
-			imageData[k] = palette[src[k]];
+		int sz = tex.nWidth * tex.nHeight;
+		if (palette && src)
+		{
+			for (int k = 0; k < sz; k++) {
+				imageData[k] = palette[src[k]];
+			}

```

Comment:
palette and src will always be defined at this point. If a texture can't be found in a wad then this code is skipped (line 170).

Reply:
Ok